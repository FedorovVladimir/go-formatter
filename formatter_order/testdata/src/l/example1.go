package main

func main() { // want "formatter_order"
}

// v - v // want "formatter_order"

// v2 - v2
// v3 - v3

var v = "v"
