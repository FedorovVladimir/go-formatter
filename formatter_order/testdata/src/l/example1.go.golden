package main

// v - v // want "formatter_order"

// v2 - v2
// v3 - v3

var v = "v"

func main() { // want "formatter_order"
}
